apps/dashboard/src/app/(dashboard)/dashboard/connect/in-app-wallets/layout.tsx

```diff
@@ -0,0 +1,5 @@
+import { Suspense } from "react";
+
+export default function Layout(props: { children: React.ReactNode }) {
+  return <Suspense fallback={null}>{props.children}</Suspense>;
```

Comment:
just for my own understanding why do we need this?

Reply:
we don't need it anymore, let me delete that